packages/smooth_app/lib/generic_lib/widgets/smooth_text_form_field.dart

```diff
@@ -78,6 +78,7 @@ class _SmoothTextFormFieldState extends State<SmoothTextFormField> {
       decoration: InputDecoration(
         prefixIcon: widget.prefixIcon,
         filled: true,
+        hintMaxLines: 2,
```

Comment:
I would even say one here

Reply:
Hi @M123-dev,
I added `hintMaxLines` so that text doesn't break in other languages because there is a possibility that may be in some languages text won't be able to fit in 1 line.

Alternative
We can add `overflow` property to the hint style.

What do you suggest?

Reply:
Yes a overflow would be good

Reply:
Changes done.